tox.ini

```diff
@@ -4,14 +4,13 @@ envlist = py27-django111,py35-django{111,22}
 [testenv]
 envdir=
     # Use the same environment for all commands running under a specific python version
-    py27: {toxworkdir}/py27
     py35: {toxworkdir}/py35
+    py38: {toxworkdir}/py38
 
 passenv =
     TEST_INTEGRATION
     TEST_DATA
 deps =
-    django111: -r requirements/django.txt
```

Comment:
This file is generated with https://github.com/eduNEXT/eox-tagging/blob/00f49d1dbae999bc6f1b81a9b6855c7cb38d23c7/Makefile#L46. We should remove that from the Makefile or rename django22.txt to django.txt and change the line after this one.

Reply:
Right! I removed it! Thank you